PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/org/jetbrains/plugins/azure/functions/run/AzureFunctionsDotNetCoreRuntime.kt

```diff
@@ -34,12 +34,17 @@ import com.jetbrains.rider.runtime.RunningAssemblyInfo
 import com.jetbrains.rider.runtime.SuspendedAttachableDotNetRuntime
 import com.jetbrains.rider.runtime.dotNetCore.DotNetCoreRuntimeType
 import org.jetbrains.plugins.azure.functions.coreTools.FunctionsCoreToolsInfo
+import org.jetbrains.plugins.azure.functions.run.localsettings.FunctionsWorkerRuntime
 
-class AzureFunctionsDotNetCoreRuntime(val coreToolsInfo: FunctionsCoreToolsInfo)
+class AzureFunctionsDotNetCoreRuntime(val coreToolsInfo: FunctionsCoreToolsInfo, val workerRuntime: FunctionsWorkerRuntime)
     : DotNetRuntime(DotNetCoreRuntimeType), SuspendedAttachableDotNetRuntime {
 
     override fun createDebugState(dotNetExecutable: DotNetExecutable, executionEnvironment: ExecutionEnvironment) : IDotNetDebugProfileState {
-        return AzureFunctionsDotNetCoreDebugProfile(dotNetExecutable, executionEnvironment, coreToolsInfo.coreToolsExecutable, coreToolsInfo.coreToolsPath)
+        return when (workerRuntime) {
+            FunctionsWorkerRuntime.DotNetDefault -> AzureFunctionsDotNetCoreDebugProfile(dotNetExecutable, executionEnvironment, coreToolsInfo.coreToolsExecutable, coreToolsInfo.coreToolsPath)
```

Comment:
`coreToolsInfo.coreToolsPath` is unused variable inside `AzureFunctionsDotNetCoreDebugProfile`.